### PR TITLE
README update

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,24 +57,17 @@ Installation
...
 
-    pip install dimod[all]
-
-To install from source:
-
-.. code-block:: bash
-
-    pip install -r requirements.txt
-    python setup.py install
+    pip install .
```
Comment (JoelPasvolsky) [inline README.rst:70]: I get an error:
```
ERROR: Could not install packages due to an OSError: [('C:\\Users\\jpasvolsky\\!git_ADTT\\test_dimod\\dimod\\docs\\LICENSE', 'C:\\Users\\jpasvolsky\\AppData\\Local\\Temp\\pip-req-build-w6ovc33b\\docs\\LICENSE', 'symbolic link privilege not held'), ('C:\\Users\\jpasvolsky\\!git_ADTT\\test_dimod\\dimod\\docs\\README.rst', 'C:\\Users\\jpasvolsky\\AppData\\Local\\Temp\\pip-req-build-w6ovc33b\\docs\\README.rst', 'symbolic link privilege not held')]
```

Comment (arcondello) [reply README.rst:70]: Incredible. Windows is why we cannot have nice things.

File: README.rst [hunk 2]
```diff
@@ -89,6 +79,35 @@ in place:
 
 .. installation-end-marker
 
+Contributing
```
Comment (JoelPasvolsky) [inline README.rst:82]: For consistency: we usually have (1) license before contrib and (2) most have "Contribution" though personally I prefer "Contributing"

File: README.rst [hunk 3]
```diff
@@ -89,6 +79,35 @@ in place:
...
+------------
+
+Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+has guidelines for contributing to Ocean packages.
+
+dimod includes some formatting customization in the
+`.clang-format <.clang-format>`_ and `setup.cfg <setup.cfg>`_ files.
+
+Reno
```
Comment (JoelPasvolsky) [inline README.rst:91]: Can we have the header "Release Notes" instead? That will be self-explanatory also to users who are not familiar with this particular package

**Review by JoelPasvolsky — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (arcondello) [reply README.rst:82]: I will do (1) but not (2)

File: README.rst (hunk 1 above)
Comment (arcondello) [reply README.rst]: For posterity, you can also do `pip install . --use-feature=in-tree-build` which will become the default behaviour in the future. For now I have reverted to the old syntax.

**Pull request merged**